proto/buf/alpha/registry/v1alpha1/repository.proto

```diff
@@ -114,6 +130,9 @@ service RepositoryService {
   rpc GetRepositorySettings(GetRepositorySettingsRequest) returns (GetRepositorySettingsResponse);
   // UpdateRepositorySettingsByName updates the settings of a repository.
   rpc UpdateRepositorySettingsByName(UpdateRepositorySettingsByNameRequest) returns (UpdateRepositorySettingsByNameResponse);
+  // GetRepositoriesMetadata gets the metadata of the repositories in the request, the length of repositories in the
+  // request should match the length of the metadata in the response, but the order in the response is unspecified.
```

Comment:
Under what conditions do we need to load metadata from multiple repositories at the same time? 

I think we do have some other API endpoints where we match the order in request/responses, we should probably do the same here as well.

Reply:
let say in a repositories list, its likely that we will want to load last commit time and owner verification status on the list, hence loading metadata from multiple repositories at the same time.

for match the order in request/responses, that was what I thought originally too, but it doesn't seems to be the case for the `GetRepositoriesByFullName`, which also mention `Response order is unspecified`. However it seems like `CodeGenerationService.GenerateCode` and `GenerateService.GeneratePlugins` have the response order matches the request order, so I am open to change it.